src/SDK/SmartDetectors.ProjectType/SmartDetectors.ProjectType.csproj

```diff
@@ -108,9 +108,8 @@
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
     <Reference Include="System" />
-    <Reference Include="System.Collections.Immutable, Version=1.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\System.Collections.Immutable.1.3.1\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="System.Collections.Immutable, Version=1.2.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\System.Collections.Immutable.1.5.0\lib\netstandard2.0\System.Collections.Immutable.dll</HintPath>
```

Comment:
>netstandard2 [](start = 73, length = 12)

I had issues with that in SDRE

Reply:
the SDRE doesn't use this project

---
In reply to: [258436851](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/50#discussion_r258436851) [](ancestors = 258436851)